app/build.gradle.kts

```
@@ -39,8 +40,9 @@ dependencies {
   implementation("androidx.appcompat:appcompat:1.2.0")
   implementation("com.google.android.material:material:1.2.1")
   implementation("androidx.constraintlayout:constraintlayout:2.0.4")
-  implementation("androidx.navigation:navigation-fragment-ktx:2.3.1")
-  implementation("androidx.navigation:navigation-ui-ktx:2.3.1")
+  implementation("androidx.navigation:navigation-fragment-ktx:2.3.2")
+  implementation("androidx.navigation:navigation-ui-ktx:2.3.2")
+  implementation("androidx.legacy:legacy-support-v4:1.0.0")
```

Нам не нужно легаси)
```suggestion
  implementation "androidx.fragment:fragment-ktx:1.2.5"
```